Header header
#message ID, r for raw tag to anchor, c for range bias corrected tag to anchor, a for range bias corrected anchor to anchor ranges
uint16 addrHorz
#tag address
uint8 countHorz
#message count in this batch
int32 distHorz
#distance in mm
int32 pdoaHorz
#phase difference of arrival in mDeg
uint16 addrVert
#tag address
uint8 countVert
#message count in this batch
int32 distVert
#distance in mm
int32 pdoaVert
#phase difference of arrival in mDeg
